submissions/chabuChabu/css/contact-styles.css

```diff
@@ -23,6 +23,10 @@ header .logo{
 	height: inherit;
 	margin-left: 20px;
 }
+<<<<<<< HEAD
```

Comment:
You have a lot on unresolved merge conflict. Locate the session of your file and remove all this `<<<< HEAD ==== >>>> 206...` Same also in other places

Reply:
Changes made